### DISCUSSION TITLE
Do I need to explicity import hardhat-network-helpers

### DISCUSSION THREAD

**Discussion by dtnwen** [NomicFoundation/hardhat#3789] [Troubleshooting] [answered]
I'm using hardhat-toolbox, I imported `require("@nomicfoundation/hardhat-toolbox");` in _hardhat.config.js_

Even though I got error 
```
ReferenceError: loadFixture is not defined
      at Context.<anonymous> (test/Faucet-test.js:14:35)
      at processImmediate (node:internal/timers:476:21)
```

Do I need to explicity import `const { loadFixture } = require('@nomicfoundation/hardhat-network-helpers');` and alwo `const { expect } = require('chai');` in my files?
![image](https://user-images.githubusercontent.com/65008502/227165889-e7ae12e2-42ca-4f2d-8b4f-7e8018c6750c.png)

**Comment by fvictorio** [comment, marked as answer]
Hi @dtnwen! This is a good question. Yes, you have to import it in each test file, and importing it in the config doesn't have any effect. The reason is that this package, unlike most other packages we have, is not a Hardhat plugin but a standalone thing.

We've considered turning it into a plugin, so that you can import in the config and have something added to the HRE (maybe `hre.helpers`). But for now you have to use it as a utils package, similar to how `chai` is imported in each test file.
